charts/incubator/deemix/questions.yaml

```diff
@@ -0,0 +1,338 @@
+# Include{groups}
+portals:
+  web_portal:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+questions:
+  - variable: portal
+    group: "Container Image"
+    label: "Configure Portal Button"
+    schema:
+      type: dict
+      hidden: true
+      attrs:
+        - variable: enabled
+          label: "Enable"
+          description: "enable the portal button"
+          schema:
+            hidden: true
+            editable: false
+            type: boolean
+            default: true
+# Include{global}
+  - variable: PUID
+    label: "PUID"
+    description: "User ID of the user you want the container to run as in order to fix folder permission issues"
+    schema:
+      type: int
+      default: 1000
+  - variable: PGID
+    label: "PGID"
+    description: "Group ID, see PUID"
+    schema:
+      type: int
+      default: 1000
+  - variable: UMASK_SET
+    label: "UMASK_SET"
+    description: "Setting UMASK for file permissions Default is 022"
+    schema:
+      type: string
+      default: "022"
+  - variable: INTPORT
+    label: "Internal Port"
+    description: "This changes the internal port of deemix. DON'T CHANGE THIS IF YOU DONT KNOW WHY YOU WOULD DO IT"
+    schema:
+      type: int
+      default: 3333
```

Comment:
This needs to be moved under `env`

Reply:
Also this is gonna change the app's port to 3333 instead of 6595...
Remove it
Or put it only on values.yaml under env 
INTPORT = "{{ .Values.service.main.ports.main.targetPort }}"